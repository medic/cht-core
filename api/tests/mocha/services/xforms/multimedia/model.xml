<root xmlns="http://www.w3.org/2002/xforms" xmlns:xf="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <model>
    <instance>
        <media xmlns:jr="http://openrosa.org/javarosa" id="multimedia">
          <meta>
            <instanceID/>
          </meta>
        </media>
      </instance>
  </model>
</root>
