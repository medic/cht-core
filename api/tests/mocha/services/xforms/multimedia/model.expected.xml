
  <model>
    <instance>
        <media xmlns:jr="http://openrosa.org/javarosa" id="multimedia">
          <meta>
            <instanceID/>
          </meta>
        </media>
      </instance>
  </model>

